geometry_msgs/Point[] point_array
Arrays[] array2d
int32 num_pts
